promotion/datastore.go

```diff
@@ -16,6 +16,8 @@ import (
 	_ "github.com/golang-migrate/migrate/v4/source/file"
 )
 
+var desktopPlatforms = [...]string{"linux", "osx", "windows"}
```

Comment:
should `"desktop"` be in there?

Reply:
platform will already be `desktop`, so it doesn't matter for the current way this list is used